src/Web/ClientApp/src/app/components/channel/overview/overview.component.ts

```diff
@@ -40,4 +40,14 @@ export class OverviewComponent implements OnChanges {
 			}
 		});
 	}
+
+	getRedirectRoute(route: string): string {
+		if (route) {
+			if (route.slice(-3) === '...') {
```

Comment:
for self-reference, this appears to be a safe call. From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice#parameters:

> If start is greater than the index range of the sequence, an empty array is returned.

So in cases where it is shorter than an ellipsis, it should jump to the `else` clause.